packages/paste-core/components/callout/src/CalloutList.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {Text, safelySpreadTextProps} from '@twilio-paste/text';
+import type {BoxProps} from '@twilio-paste/box';
+
+type AsTags = 'ul' | 'ol';
+
+export interface CalloutListProps extends Partial<Omit<HTMLLIElement, 'children'>> {
```

Comment:
question: was `HTMLLIElement` intended for extension?

Reply:
oop yeah its supposed to be HTMLElement